MANIFEST.in

```diff
@@ -16,6 +16,7 @@ recursive-include tools *.py *.sh
 recursive-include mdn *.py
 recursive-include mdn/jinja2 *.html
 recursive-include mdn/static *.css
+recursive-include requirements *.txt
```

Comment:
I may need to move it to /requirements, yeah. It is for readthedocs, which installs it's own preferred version of Sphinx, etc.  If it tries to install a package with a C-dependency (like psycopg), then the docs don't build.  I'll probably have to do a two-phase move - add an empty file in this PR, update the RTD config, remove requirements-rtd.txt in the next PR.